rackcity/views/asset_views.py

```diff
@@ -413,8 +433,23 @@ def asset_modify(request):
             },
             status=HTTPStatus.BAD_REQUEST
         )
+    if not user_has_asset_permission(
```

Comment:
Ditto

Reply:
same as above, there's no way to check for permissions earlier because you need to be able to access the datacenter